db/knexfile.js

```diff
@@ -0,0 +1,53 @@
+// Update with your config settings.
```

Comment:
I don't see any tests.  Have you written any yet?

Reply:
Have not written any yet. That is the goal by end of Saturday.